app/uk/gov/hmrc/incorporatedentityidentification/models/RegisterWithMultipleIdentifiersModel.scala

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.incorporatedentityidentification.models
+
+import play.api.libs.json.{Json, OFormat}
+
+case class RegisterWithMultipleIdentifiersModel(company: CompanyModel)
```

Comment:
I don't think we need this model, it doesn't need to be wrapped for this use case like on VAT.